src/components/Header/Header.tsx

```diff
@@ -0,0 +1,70 @@
+import cn from 'classnames';
+import { useEffect, useRef, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { ErrorType } from '../../types/ErrorType';
+
+interface Props {
+  todosFromServer: Todo[];
+  createNewTodo: (title: string) => Promise<void>;
+  errorFound: (error: ErrorType) => void
+}
+
+export const Header = ({
+  todosFromServer,
+  createNewTodo,
+  errorFound,
+}: Props) => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [todoTitleDisabled, setTodoTitleDisabled] = useState(false);
+  const todoTitleRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    todoTitleRef.current?.focus();
+  }, [todoTitleDisabled]);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const title = todoTitle.trim();
+
+    if (!title) {
+      errorFound(ErrorType.TitleEmpty);
+
+      return;
+    }
+
+    setTodoTitleDisabled(true);
+    createNewTodo(title)
+      .then(() => setTodoTitle(''))
+      .catch(() => {})
```

Comment:
You should set an error message and show to user it, so they could know that something wrong

Reply:
It's already implemented inside createNewTodo